CONTRIBUTING.md

```diff
@@ -11,12 +11,25 @@ Contributing doesn't necessarily mean commiting code, we also encourage you to:
 - Help write documentation
 - Use the Design System in your project and provide feedback
 
+Highlight topics:
+
+* [Development](#development)
```

Comment:
I think this might make more sense as a table of contents at the top of the file